src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -50,6 +50,25 @@ export default class SimidMapCreative extends BaseSimidCreative {
    * @private 
   */
   grantLocationAccess_() {
-    //ToDo(kristenmason@): implement map
+    this.loadMap_();
+  }
+
+  /**
+   * Loads a map object that currently displays a hardcoded location.
+   * @param {Object} coordinates - the LatLng object of user's current location
+   * ToDo(kristenmason@): implement grant location access and modify
```

Comment:
Is there a better way to make these stand out so that they won't be missed/forgotten? Maybe add a github issue and link it here so that the url is obvious